api/apps/graphql/resolvers.js

```diff
@@ -93,3 +95,45 @@ export const authors = (root, args, req, ast) => {
     })
   })
 }
+
+export const relatedArticlesPanel = (root) => {
+  const tags = root.tags || null
+  const args = {
+    omit: [ObjectId(root.id)],
+    published: true,
+    featured: true,
+    channel_id: ObjectId(root.channel_id),
+    tags: tags,
+    limit: 3,
+    sort: '-published_at'
+  }
+  return new Promise((resolve, reject) => {
+    mongoFetch(_.pick(args, _.identity), (err, results) => {
+      if (err) {
+        reject(new Error(err))
```

Comment:
FYI it's typically good to do `return reject(new Error(err))` here because to prevent the code below from executing. Although this is not [necessary in this case](https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject), just FYI. I won't block this PR for it, b/c if you do want to address this I'd prefer updating `mongoFetch` to return a promise allowing async/await so you can do...

````javascript
export const relatedArticlesPanel = async (root) => {
  const args = { ... }
  const { results } = await mongoFetch(...)
  return results
}
````

Reply:
Sounds good, I'll follow up with the modernized `mongoFetch` next!